client/src/index.js

```diff
@@ -1,12 +1,12 @@
 import { createRoot } from "react-dom/client";
-import { BrowserRouter } from "react-router-dom";
+import { BrowserRouter ,Route} from "react-router-dom";
```

Comment:
Hi Christian, the routes were already created in the app.js file so I dont think it needs to be changed here

Reply:
Maybe Charles could advise